src/util/ReactUtils.ts

```diff
@@ -282,18 +207,22 @@ export const isValidSpreadableProp = (
   property: unknown,
   key: string,
   includeEvents?: boolean,
-  svgElementType?: FilteredSvgElementType,
+  svgElementType?: SVGElementType,
 ) => {
   /**
    * If the svg element type is explicitly included, check against the filtered element key map
    * to determine if there are attributes that should only exist on that element type.
    * @todo Add an internal cjs version of https://github.com/wooorm/svg-element-attributes for full coverage.
    */
-  const matchingElementTypeKeys = FilteredElementKeyMap?.[svgElementType] ?? [];
+  const svgElementAttributesInCamelCase = getSVGElementAttributesInCamelCase(svgElementAttributes);
+  const matchingElementTypeKeys = svgElementAttributesInCamelCase?.[svgElementType] ?? [];
+
+  const { polygon, polyline, svg, ...rest } = svgElementAttributesInCamelCase;
```

Comment:
why do these three need to be extracted out?

Reply:
this was me, messing with things!